src/components/LogViewer/index.scss

```diff
@@ -0,0 +1,262 @@
+// TODO: Figure out if this can be moved up without breaking anyone
```

Comment:
This should be changed to similar strategy used for Firestore viewer. https://github.com/firebase/firebase-tools-ui/blob/03d0695d684c0ea05de2fa1ce5cf6744c3579267/src/components/Firestore/index.scss#L24-L28

Also FYI, the style here is effectively global, despite it's in a directory. We don't have CSS modules or shadow DOM or anything like those

Reply:
In particular, we should set those heights on the outmost container (`#log` etc.), not on `.App-main`.

Reply:
Imma leave this for a second PR so we can get this one in.